tests/cypress/e2e/print.test.js

```diff
@@ -0,0 +1,25 @@
+describe('Publish a new print issue', () => {
+	before(() => {
+		cy.login();
+	});
+
+	it("Open new print issue page", () => {
+		cy.visit(`/wp-admin/post-new.php?post_type=print-issue`);
+		cy.get("#title").type("Print Title 1");
+		cy.get("#pi-section-add").click();
+		cy.get("#pi-section-name").type("Section title 1");
+		cy.get("#pi-section-add-confirm").click();
+		cy.wait(1000)
```

Comment:
@iamdharmesh those waits are needed as those buttons are conditionally loaded based on the ajax and JS code, so we need to wait for it to appear.

Reply:
@dhanendran `.should('exist')` waits up to 4 secs to check it button exists. Example: [Check this](https://github.com/10up/cypress-wp-utils/blob/09361aef3b5a1640de36964baa1996099b5e5821/src/commands/upload-media.ts#L18-L20)
Also,  for the case of ajax you can also intercept the request and wait for the request to complete. Example: [Check this](https://github.com/10up/cypress-wp-utils/blob/trunk/src/commands/create-post.ts#L102-L110). fixed wait generates flaky tests, so better to avoid them where possible.

Reply:
@iamdharmesh this is ready now.